tests/models/ensemble/bidaf_ensemble_test.py

```diff
@@ -0,0 +1,105 @@
+# pylint: disable=no-self-use,invalid-name,line-too-long
```

Comment:
I don't think you need to disable `line-too-long` here.  The lines below could be shortened dramatically, and it would help the readability of this test.